nexus/db-queries/src/db/datastore/vpc.rs

```diff
@@ -650,8 +650,8 @@ impl DataStore {
         // Resolve each VNIC in the VPC to the Sled it's on, so we know which
```

Comment:
(This belongs on L643 -- thanks GitHub)

This function returns the list of `Sled`s hosting instances _or control plane zones_ with network interfaces on the provided VPC, right?

Reply:
Correct; updated comment in 4fa423bf0